UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -72,16 +85,29 @@ public void AfterTest(ITest test)
 
                 _denyExecutionSynchronizationContext?.ThrowIfSwitchOccurred();
             }
+            catch (Exception ex) when (_threadException != null)
+            {
+                // ignore the follow-up exception
+                LoggingService.Log($"{nameof(AfterTest)} ignoring", ex);
+            }
             finally
             {
+                AppSettings.CheckForUpdates = _checkForUpdatesOriginalValue;
                 Application.ThreadException -= HandleApplicationThreadException;
-                _threadException?.Throw();
+                if (_threadException != null)
+                {
+                    LoggingService.Log($"{nameof(AfterTest)} rethrow", _threadException.SourceException);
+                    _threadException.Throw();
+                }
             }
         }
 
         private void HandleApplicationThreadException(object sender, ThreadExceptionEventArgs e)
         {
-            if (_threadException == null)
+            bool ignore = e.Exception.GetType() == typeof(ThreadAbortException);
```

Comment:
❗️ `ThreadAbortException` is a bug by definition. If one gets thrown, the test must fail.

Reply:
Ah, that's a remainder of the background thread in FormBrowse, which you have replaced in the meantime. This thread was aborted intentionally.